portfolio/src/main/webapp/style.css

```diff
@@ -87,8 +87,11 @@ nav,
 }
 
 /* Warning */
+#invalid-max-comments,
```

Comment:
Maybe instead of having separate ids for each, you could just have an "error" style class that you use for all of them? (or are there other differences between these that I'm not seeing?)

Reply:
Got it! They look the same. I'll use one rule to add style to all of them